data/tokens.json

```diff
@@ -2797,6 +2791,10 @@
       }
     },
     "Banner": {
+      "gtmds_banner-dismiss-button_background-color-hover": {
+        "value": "rgba({Base Notifications.gtmds_notification_color},0.1)",
```

Comment:
wow, does this thing work :D ?

Reply:
the plugin supports the css functions so in theory, yes it should work out of the box without extra effort on our end